### DISCUSSION TITLE
Generate overlay from files diff

### DISCUSSION THREAD

**Discussion by dudicoco** [k14s/ytt#921] [Q&A]
Some tools can generate a json patch from a diff of two files, for example https://github.com/josephburnett/jd.

The patch can then be applied to a different file.

Is it possible to generate an overlay from a diff?

**Comment by prembhaskal** [comment]
@cppforlife can tell more about this.

@dudicoco if you could post this question on slack, you will get faster response.

**Reply by dudicoco** [reply]
Thanks @prembhaskal.
Why can I not get a response within the discussion? Isn't that the purpose of the discussions feature? To share knowledge?

**Reply by prembhaskal** [reply]
sure, you can get response , that's why i tagged Dmitriy. But on slack channel, many ytt experts (apart from maintainers) are active, who might not see github notifications, so the suggestion.
